com.unity.render-pipelines.high-definition/Runtime/Compositor/CompositionLayer.cs

```diff
@@ -210,39 +215,47 @@ public void Init(string layerID = "")
                 m_Camera = CompositionManager.GetSceceCamera();
             }
 
+            var compositor = CompositionManager.GetInstance();
+
             // Create a new camera if necessary or use the one specified by the user
             if (m_LayerCamera == null && m_OutputTarget == OutputTarget.CameraStack)
             {
-
-                // Clone the camera that was given by the user. We avoid calling Instantiate because we don't want to clone any other children that might be attachen to the camera 
-                var newCameraGameObject = new GameObject("Layer " + layerID)
-                {
-                    hideFlags = HideFlags.HideInInspector | HideFlags.HideInHierarchy | HideFlags.HideAndDontSave
-                };
-                m_LayerCamera = newCameraGameObject.AddComponent<Camera>();
-                newCameraGameObject.AddComponent<HDAdditionalCameraData>();
-                CopyInternalCameraData();
-
-                m_LayerCamera.name = "Compositor" + layerID;
-                m_LayerCamera.gameObject.hideFlags = HideFlags.HideInInspector | HideFlags.HideInHierarchy | HideFlags.HideAndDontSave;
-                if(m_LayerCamera.tag == "MainCamera")
+                if (!compositor.IsThisCameraShared(m_Camera))
```

Comment:
The only change here is the 3-line if statement that makes the clone only if the camera was shared. But the indentation of the code has changed, so github marked everything as changed! Not sure if it was possible to create a cleaner diff somehow.

Reply:
ok no worry